data/sql_updates/alter_read_only_web_role.sql

```diff
@@ -3,8 +3,8 @@ $body$
 BEGIN
    IF NOT EXISTS (
       SELECT *
-      FROM   pg_catalog.pg_user
-      WHERE  usename = 'webro') THEN
+      FROM   pg_roles
+      WHERE  rolname = 'webro') THEN
```

Comment:
Perhaps something changed about the pg_catalog tables between minor versions? I know Postgres has been moving to the concept of roles subsuming the concept of users and groups.

That change looks fine to me.